hello/hello_search_attributes.py

```diff
@@ -28,23 +31,26 @@ async def main():
         # While the worker is running, use the client to start the workflow.
         # Note, in many production setups, the client would be in a completely
         # separate process from the worker.
+        custom_keyword_field = SearchAttributeKey.for_keyword("CustomKeywordField")
```

Comment:
Could even make this a global if you'd like and reuse it in the workflow too. At the very least, define it at the top of this function instead of redefining within the function.